src/components/Notify/NotifyForm.tsx

```diff
@@ -33,41 +88,202 @@ const NotifyForm = () => {
+                              ? item?.username?.charAt(0)
+                              : item?.first_name?.charAt(0)}{' '}
+                            {item?.last_name?.charAt(0)}
+                          </Text>
+                        </View>
+                      )}
+                      <Text style={styles.userNameDropDown}>
+                        {item?.username
+                          ? item?.username
+                          : item?.first_name + item?.last_name}
+                      </Text>
+                    </TouchableOpacity>
+                  );
+                }}
+              />
+            )}
+          </View>
+        ) : null}
+      </View>
+      <View style={{ marginTop: 16 }}>
+        <TouchableOpacity
+          style={[styles.button, { backgroundColor: Colors.Primary_Color }]}
+          onPress={handleButtonPress}
+        >
+          <Text style={{ color: 'white' }}>{'Notify'}</Text>
+        </TouchableOpacity>
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
+    // flex: 1,
     padding: 16,
   },
   label: {
+    color: 'black',
     fontSize: 16,
     marginBottom: 8,
   },
   input: {
+    color: 'black',
     height: 40,
-    borderColor: 'gray',
+    borderColor: '#ccc',
     borderWidth: 1,
     marginBottom: 16,
     paddingHorizontal: 8,
   },
+  picker: {
+    width: '100%',
+    height: 50,
+    color: 'black',
+  },
+
+  pickerContainer: {
+    borderWidth: 1,
+    borderColor: '#ccc',
+    borderRadius: 2,
+    // overflow: 'hidden',
+  },
+  button: {
+    backgroundColor: '#3498db',
+    padding: 10,
+    borderRadius: 5,
+    alignItems: 'center',
+  },
+  dropDownSelector: {
+    padding: 10,
+    borderRadius: 8,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  dropDownIcon: {
+    width: 20,
+    height: 20,
+  },
+  dropDownArea: {
+    height: scale(250),
+    borderWidth: 0.5,
+    marginTop: 10,
+    borderRadius: 8,
+  },
+  userNameDropDown: {
+    padding: 20,
+    borderBottomColor: 'white',
+    width: '90%',
+    alignSelf: 'center',
+    color: StyleConfig.colors.darkGrey,
+  },
+  userDetails: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginLeft: 10,
+  },
+  userImageDropDown: {
+    width: 50,
+    height: 50,
+    borderRadius: 50,
+  },
+  defaultImageContainer: {
+    width: 50,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 50,
+    backgroundColor: StyleConfig.colors.placeholderText,
+    borderRadius: 50,
+  },
+  error: {
+    color: 'red',
+    fontSize: 10,
+  },
+  searchBar: {
+    marginBottom: scale(20),
+    marginHorizontal: 5,
+    borderBottomWidth: 0.5,
+    fontSize: scale(12),
+    color: StyleConfig.colors.darkGrey,
+  },
+  inputStyle: {
+    padding: scale(10),
+    borderRadius: 8,
+    backgroundColor: StyleConfig.colors.whiteInput,
+    fontSize: scale(12),
```

Comment:
- do we just scale the font to set a font size or is there another way to handle this?